SYCL/ESIMD/regression/copyto_char_test.cpp

```diff
@@ -54,7 +52,7 @@ int main(int, char **) {
     DataT *ref_data_for_fill_ptr = ref_data_for_fill.data();
     DataT *const out_ptr = result.data();
 
-    cgh.single_task<class Kernel>([=]() SYCL_ESIMD_KERNEL {
+    cgh.single_task<T1>([=]() SYCL_ESIMD_KERNEL {

```

Comment:
Non-blocking comment: The explicit specifying of the kernel name is not required anymore as -fsycl-unnamed-lambda is enabled by default. Removing it would also eliminate the need in 'typename T1' parameter of the test_to_copy() function.